frontend/packages/kubevirt-plugin/src/hooks/use-support-modal.ts

```diff
@@ -15,6 +15,7 @@ export const useSupportModal = (): SupportModalFunction => {
     (template, onConfirm) => {
       const isUpstream = window.SERVER_FLAGS.branding === 'okd';
       const templateSupport = getTemplateSupport(template);
+      const isCommonTemplate = template.metadata.namespace === 'openshift';
```

Comment:
we already have an `isCommonTemplate`:
https://github.com/openshift/console/blob/97ac09eeb3c4833cfaa68006d6f9a77cca70e66e/frontend/packages/kubevirt-plugin/src/selectors/vm-template/basic.ts#L31